spec/deprecation_toolkit_env.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
```

Comment:
Can you add a few lines on usage to the Testing section of the README?

Reply:
@giladshanan added to README!